examples/ex01_data/example_data_01.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Example of data analysis of contaminant data from Griftpark, Utrecht
+using mibipret 
+
+@author: Alraune Zech
+"""
+
+import sys
+# import numpy as np
+# import pandas as pd
+# import os
+
+path = '/home/alraune/GitHub/MiBiPreT/mibipret'
+
+sys.path.append(path) # append the path to module
```

Comment:
It should not be necessary to append the package path to your system path if you followed the installation instructions and installed mibipret with pip.